### ISSUE TITLE
A separate sort setting for files and for folders

### ISSUE THREAD

**Issue by WhiteMac** [nothingislost/obsidian-bartender#24]
I have made this suggestion on the Obsidian forums; copying it here as I fiercely hope it gets picked up. I think it would elevate this plugin from amazing to almost perfect.

In my case—and I believe this would make sense for a lot of users—I just want to manually sort my buckets. The folders. Within my “sorted just right” buckets, I’d just like to have the files sorted the traditional way. That is, interchangeably by modification date, creation date, or alphabetically, but rarely if ever manually.

Thank you very much for this plugin!

**Comment by looneyapache** [comment]
![image](https://user-images.githubusercontent.com/10562851/161467835-4642f2f9-b2bb-47fb-b534-fc220fd1caef.png)

Isn't this already there (if I am not mistaken)

**Comment by WhiteMac** [comment]
> ![image](https://user-images.githubusercontent.com/10562851/161467835-4642f2f9-b2bb-47fb-b534-fc220fd1caef.png)
> 
> Isn't this already there (if I am not mistaken)

No, I was suggesting a separate set of those sort settings for files and for folders. Because I'd like my folders (buckets) arranged custom, but the files within the folders arranged traditionally, either by name, creation date or modification date.

And as I said, I think many people would like that. Just like in something like Apple Mail you might have custom arranged folders, but the messages themselves are not custom, but sorted by date.

**Comment by kzhovn** [comment]
I'd love something like this - I never want folders below files, but do want to be able to custom sort my files.

(@nothingislost , thank you _so_ much for this plugin, it has been a revelation.)

**Comment by NB574** [comment]
Also would love something like this, I have certain folders which just work better my for eyes if their files are listed alphabetically. To be fair though, the joy of being able to actually custom sort folders (and have my kanban above all folders in the vault) far outweighs this, so I happily reorder my files manually if needed. This plugin is a simple QOL one which makes a huge difference for me, many thanks.
